BeeJet.Web/BotService.cs

```diff
@@ -11,19 +12,18 @@ public class BotService : BackgroundService
 
         public bool IsRunning { get; private set; }
 
-        public BotService(ILogger<BotService> logger, IConfiguration configuration)
+        public BotService(ILogger<BotService> logger, IConfiguration configuration, IBeeJetRepository repository)
         {
             _logger = logger;
             var options = new BeeJetBotOptions()
             {
                 SteamAPIKey = configuration["STEAM_KEY"],
                 IDGBClientId = configuration["IGDB_CLIENTID"],
                 IDGBClientSecret = configuration["IGDB_SECRET"],
-                DiscordToken = configuration["DISCORD_TOKEN"]
+                DiscordToken = configuration["DISCORD_TOKEN"],
+                SteamSignInLink = configuration["SteamSignIn"]
```

Comment:
Misschien voor de consistentie, deze key renamen naar "STEAM_SIGNINURL" of iets?